examples/nextjs-todo/components/ui/TodoForm.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useEffect } from "react";
+import { TBaseParameter, TUnit } from "@arkejs/client";
+import { Form, FormField } from "@arkejs/form";
+import useClient from "@/arke/useClient";
+import { Button, Input } from "@arkejs/ui";
+
+import { Dialog } from "@arkejs/ui";
+
+type ID = string;
+export interface CrudState {
+  add?: boolean;
+  edit?: boolean | ID;
+  delete?: boolean | ID;
+}
+
+export default function TodoForm({ onClose, onSubmit, getTodosData }: any) {
```

Comment:
add `active` prop as `string | undefined`. This prop will be used below instead of `onSubmit.id`